python/llm/src/bigdl/llm/transformers/linear_quant.py

```diff
@@ -153,13 +153,13 @@ def ggml_matmul_src1_x_src0_t(src0: torch.Tensor,
     if src1.dtype != torch.float32:
         src1 = src1.float()
 
-    src0_ptr = src0.data_ptr() + (src0.storage_offset() * src0.element_size())
-    src1_ptr = src1.data_ptr() + (src1.storage_offset() * src1.element_size())
```

Comment:
I think the address returned by `.data_ptr()` is always right, even this tensor is using others tensor's `Storage`, so there is no need to add `storage_offset * element_size`.